test/e2e/user_management_test.go

```diff
@@ -255,12 +298,124 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
+				ConditionSet(Default(), ApprovedAutomatically(), Deactivating())...))
+			require.NoError(s.T(), err)
+
+			// Verify resources have been provisioned
+			VerifyResourcesProvisionedForSignup(t, s.hostAwait, userSignup, "base", s.memberAwait)
+
+			s.T().Run("user set to deactivated after deactivating", func(t *testing.T) {
+				// Set the provisioned time even further back
+				tierDeactivationDuration := time.Duration(baseTier.Spec.DeactivationTimeoutDays+4) * time.Hour * 24
+				mur, err = s.hostAwait.UpdateMasterUserRecordStatus(mur.Name, func(mur *toolchainv1alpha1.MasterUserRecord) {
+					mur.Status.ProvisionedTime = &metav1.Time{Time: time.Now().Add(-tierDeactivationDuration)}
+				})
+				require.NoError(s.T(), err)
+				s.T().Logf("masteruserrecord '%s' provisioned time adjusted to %s", mur.Name,
+					mur.Status.ProvisionedTime.String())
+
+				// Set the LastTransitionTime of the status to 3 days in the past
+				deactivatingLastTransitionTime := metav1.Time{Time: time.Now().Add(time.Duration(-3) * time.Hour * 24)}
+
+				// Update the LastTransitionTime of the DeactivatingNotificationCreated condition
+				newConditions := make([]toolchainv1alpha1.Condition, len(userSignup.Status.Conditions))
+				copy(newConditions, userSignup.Status.Conditions)
+				for i, c := range newConditions {
+					if c.Type == toolchainv1alpha1.UserSignupUserDeactivatingNotificationCreated {
+						newCondition := toolchainv1alpha1.Condition{
+							Type:               c.Type,
+							Status:             c.Status,
+							LastTransitionTime: deactivatingLastTransitionTime,
+							Reason:             c.Reason,
+							Message:            c.Message,
+							LastUpdatedTime:    c.LastUpdatedTime,
+						}
+						newConditions[i] = newCondition
+						break
+					}
+				}
+				userSignup.Status.Conditions = newConditions
+
+				// Confirm that the LastTransitionTime has been correctly set
+				updated, found := condition.FindConditionByType(userSignup.Status.Conditions,
+					toolchainv1alpha1.UserSignupUserDeactivatingNotificationCreated)
+				require.True(t, found)
+				require.Equal(t, deactivatingLastTransitionTime, updated.LastTransitionTime)
+
+				// Save the updated UserSignup's Status
+				require.NoError(t, s.hostAwait.Client.Status().Update(context.TODO(), userSignup))
+
+				// Trigger a reconciliation of the deactivation controller by updating the MUR
+				// - The SyncIndex property of the UserAccount is intended for the express purpose of triggering
+				//   a reconciliation, so we set it to some new unique value here
+				mur.Spec.UserAccounts[0].SyncIndex = uuid.Must(uuid.NewV4()).String()
+				require.NoError(t, s.hostAwait.Client.Update(context.TODO(), mur))
+
+				// The user should now be set to deactivated
+				userSignup, err = s.hostAwait.WaitForUserSignup(userSignup.Name,
+					wait.UntilUserSignupHasConditions(ConditionSet(ApprovedAutomatically(), Deactivated())...))
+				require.NoError(s.T(), err)
```

Comment:
Maybe also worth checking/waiting for the MUR to be deleted? Or it's already covered in other tests?

Reply:
It's covered in other tests, but I think it's a good idea to confirm here also since it is supposed to be a full deactivation lifecycle test.  Fixed.